core/App/components/modals/NetInfoModal.tsx

```diff
@@ -26,13 +28,13 @@ const NetInfoModal: React.FC<NetInfoModalProps> = ({ visible, onSubmit = () => n
   return (
     <Modal visible={visible} transparent={true}>
```

Comment:
No longer require the wrapping modal

Reply:
If I don't use the modal then the background looks funky
![image](https://user-images.githubusercontent.com/36937407/186546927-f094b9e2-67e1-486a-8111-b0801dd02789.png)